Dockerfile

```diff
@@ -1,13 +1,21 @@
-FROM node:9.2.0-alpine
+FROM node:9-slim
 
-ENV SCREENIE_VERSION=1.2.0
+ENV SCREENIE_VERSION=2.0.0
+ENV SCREENIE_CHROMIUM_ARGS="--no-sandbox"
 
-RUN apk add --no-cache fontconfig curl ca-certificates \
-    && curl -Ls "https://github.com/dustinblackman/phantomized/releases/download/2.1.1/dockerized-phantomjs.tar.gz" | tar xz -C / \
-    && npm install -g screenie-server@$SCREENIE_VERSION --unsafe-perm \
-    && apk del curl \
-    && rm -rf /tmp/*
+RUN apt-get update && \
+    apt-get install -yq gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 \
```

Comment:
I should also add `ttf-ancient-fonts` in otrder to get uncode support on basic fonts.